Apps/phpassivetotal/exclude_files.txt

```diff
@@ -0,0 +1,3 @@
+whitesource*
```

Comment:
this exclude file is no longer necessary as they have been added to the global excludes list: https://cd.splunkdev.com/phantom/app_release/-/blob/master/data/global_build_file_excludes.json

if we need to add new excludes they should generally be added there instead of per-repo